outport/firehose/firehoseIndexerTxPool.go

```diff
@@ -78,7 +79,8 @@ func getTxs(txs map[string]data.TransactionHandlerWithGasUsedAndFee) (map[string
 			return nil, fmt.Errorf("%w, hash: %s", errCannotCastTransaction, txHash)
```

Comment:
we can use hex encoded hash here in error
- available also for the next 4 functions

Reply:
Right, good suggestion.
Changed them all.